node/src/effect.rs

```diff
@@ -1096,6 +1096,24 @@ impl<REv> EffectBuilder<REv> {
         .await
     }
 
+    /// Gets the requested sufficient signatures for a given block hash.
```

Comment:
We should include in the comment what exactly _sufficient_ means. Personally, I think it's a bit weird that the caller requests _sufficient_ but the definition of what it means is in the storage.

Reply:
Added more explanatory comments in c51cb271017b50715bca4489f250303a0620b065